### PR TITLE
Update publish instructions

### PR DESCRIPTION
**Pull request by JordanMartinez**

### CODE REVIEW

**Comment by JordanMartinez** [comment]
Fixes #402

File: static/help-docs/authors.md
Comment (thomashoneyman) [inline static/help-docs/authors.md:8]:
```suggestion
- A tag points to the same commit that is currently checked out (whether by a checked out commit, branch, or tag)
```

**Comment by JordanMartinez** [comment]
I'll wait until I get an approval here before redeploying Pursuit.

**Review by thomashoneyman — APPROVED** [review]

**Pull request merged**
